scripts/release_scripts/repo_specific_changes_fetcher.py

```diff
@@ -29,7 +29,7 @@
 # TODO(#15567): The order can be fixed after Literal in utils.py is loaded
 # from typing instead of typing_extensions, this will be possible after
 # we migrate to Python 3.8.
-from scripts import common  # isort:skip  # pylint: disable=wrong-import-position
+from scripts import common  # isort:skip  ng-import-position
```

Comment:
Why this change? it looks like a mistake

Reply:
Done